web/src/message_lists.ts

```diff
@@ -0,0 +1,84 @@
+import $ from "jquery";
+import assert from "minimalistic-assert";
+
+import * as blueslip from "./blueslip";
+import * as inbox_util from "./inbox_util";
+import type {Message} from "./message_store";
+import * as recent_view_util from "./recent_view_util";
+import * as ui_util from "./ui_util";
+
+// TODO(typescript): Move this to message_list_view when it's
+// converted to typescript.
+type MessageListView = {
+    update_recipient_bar_background_color: () => void;
+    _render_win_start: number;
+    _render_win_end: number;
+};
+
+type MessageList = {
+    table_name: string;
+    view: MessageListView;
+    selected_id: () => number | undefined;
+    selected_row: () => JQuery | undefined;
```

Comment:
```suggestion
    selected_row: () => JQuery;
```

The error return is `$(undefined)`, which is a valid `JQuery` object with `.length === 0`.

https://github.com/zulip/zulip/blob/68f2535ebca80540a013c41454614cf560646188/web/src/message_list_view.js#L1448-L1452